Cargo.toml

```diff
@@ -14,5 +14,4 @@ members = [
 lto = "thin"
 
 [patch.crates-io]
-# TODO(Zahari): switch released version once TLS protocol support is out.
-linkerd2-proxy-api = { git = 'https://github.com/linkerd/linkerd2-proxy-api', rev = 'c5648ae2a1e405cc6b8aca20522356ebdf20f1ea' }
\ No newline at end of file
+linkerd2-proxy-api = { git = "https://github.com/linkerd/linkerd2-proxy-api.git", branch = "alpeb/rate-limiting" }
```

Comment:
Pulled out the API bump into a separate PR https://github.com/linkerd/linkerd2/pull/13290